Riot/Modules/Common/Recents/RecentsViewController.m

```diff
@@ -29,7 +29,7 @@
 
 #import "GeneratedInterface-Swift.h"
 
-@interface RecentsViewController () /*<CreateRoomCoordinatorBridgePresenterDelegate, RoomsDirectoryCoordinatorBridgePresenterDelegate, RoomNotificationSettingsCoordinatorBridgePresenterDelegate>*/
+@interface RecentsViewController () <CreateRoomCoordinatorBridgePresenterDelegate, RoomsDirectoryCoordinatorBridgePresenterDelegate, RoomNotificationSettingsCoordinatorBridgePresenterDelegate>
```

Comment:
Please let comment `CreateRoomCoordinatorBridgePresenterDelegate` and `RoomsDirectoryCoordinatorBridgePresenterDelegate`

Reply:
Fixed in latest commits